asapdiscovery-data/asapdiscovery/data/fitness.py

```diff
@@ -64,14 +67,19 @@ def apply_bloom_abstraction(fitness_dataframe: pd.DataFrame) -> dict:
             total count (~confidence)
         ]
     """
+    # add this column in case we're pulling in an experiment that has different data. We need to find
```

Comment:
We should pre-process so that it is in a consistent format, perhaps the submitting labs may not have a common language, but we should to avoid an explosion of complexity or inconsistent data that looks consistent.

Reply:
Raised issue #649

Reply:
agreed - imo fine for now, let's revisit the next time we receive a divergent format so that we can generalize between the three.